KsApi/fragments/UserFragment.graphql

```diff
@@ -1,7 +1,22 @@
 fragment UserFragment on User {
```

Comment:
Ah I didn't notice that but it definitely will have to be addressed eventually. Going to flag this to Paulo to put in another ticket because this PR is way too large already.